OFEP-flagd-grpc-sync.md

```diff
@@ -0,0 +1,38 @@
+## OFEP: Add gRPC sync support to flagd
+
+## State: DRAFTING
```

Comment:
Reminder to update state before merging.

Reply:
Done